### ISSUE TITLE
Not extracting imported CSS

### ISSUE THREAD

**Issue by jpkontreras** [egoist/vbuild#156]
Hello. Im trying to extract the css . But when i try the vbuild command,  the css gets appended into the js file. I even downloaded the css loader but nothing happens. 

vbuild.config.js
```
module.exports = {
    vendor: false,
    autoprefixer: {
        browsers: ['ie > 8', 'last 3 versions']
    },
    filename: {
        js: 'wi.js',
        css: 'wi.css',
        static: 'static/[name].[ext]',
        chunk: '[id].chunk.js'
    }

};

```

package.json

```
 "devDependencies": {
    "babel-helper-vue-jsx-merge-props": "^2.0.2",
    "babel-plugin-syntax-jsx": "^6.18.0",
    "babel-plugin-transform-vue-jsx": "^3.4.2",
    "babel-preset-es2015": "^6.24.1",
    "css-loader": "^0.28.0",
    "less": "^2.7.2",
    "less-loader": "^4.0.3",
    "node-sass": "^4.5.2",
    "sass-loader": "^6.0.3",
    "webpack": "^2.4.1"
  },
  "dependencies": {
    "semantic-ui": "^2.2.10"
  }
```

template Vue
```
<template>
    <div>
        <button class="ui button positive">
            <slot></slot>
        </button>
    </div>
</template>
<script>
    import 'semantic-ui/dist/components/button.min.css'
    export default {
        data(){
            return {
                hello: 'Hello World!'
            }
        }
    }
</script>
```

**Comment by jpkontreras** [comment]
I also installed extract text like u said in #82 . This is the output on my console. 
![](https://i.gyazo.com/473e6ef60756609507a4d76e53563086.png)

**Comment by egoist** [comment]
Fixed in `v7.2.4`, btw #82 is a pretty old issue
